README.md

```diff
@@ -347,7 +384,19 @@ _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from
      target: control-plane
    ```
 
-2. Verify that the module creation succeeded and observe the `mod` folder. It will contain a `component-descriptor.yaml` with a definition of local layers.
+   **[DEPRECATED] Legacy Module Build and Push**
```

Comment:
I suggest we remove this part, this `--kubebuilder-project` will be deprecated soon, not necessary to mention it.